lua/rustaceanvim/server_status.lua

```diff
@@ -7,8 +7,13 @@ local _ran_once = {}
 
 ---@param result rustaceanvim.internal.RAInitializedStatus
 function M.handler(_, result, ctx, _)
-  if result and result.health and result.health ~= 'ok' then
-    vim.notify_once("rust-analyzer health is not OK. Run 'RustLsp logFile' for details", vim.log.levels.WARN)
+  if result and result.health and result.health ~= 'ok' and result.quiescent then
+    local message = "rust-analyzer health is not OK. Run 'RustLsp logFile' for details: " .. (result.message or "empty message")
+    local severity = vim.log.levels.WARN
+    if result.health ~= 'warning' then
+      severity = vim.log.levels.ERROR
```

Comment:
issue: I deliberately chose `vim.log.levels.WARN` for all non-OK statuses, because an `ERROR` notification by default blocks the UI until the user has confirmed. 
They should be used sparingly, for example when a user command has failed or on an invalid config.

Reply:
LazyVim uses a plugin for the notification backend. That's not the default behaviour of Neovim.

> Maybe we add "error" or "warning" to the message and use levels.WARN?

Sure, that sounds reasonable.